Jenkinsfile.release

```diff
@@ -25,7 +25,7 @@ pipeline {
       kubernetes {
         label 'jenkins-release'
         defaultContainer 'jnlp'
-        yaml """
+        yaml '''
```

Comment:
Always use single quotes unless you specifically intend to do Groovy interpolation (`GString`s).

Reply:
ack